app/2.0/docs/devguide/dom-template.md

```diff
@@ -65,42 +61,32 @@ except for testing. For caveats about defining elements in the main document,
 see [main document definitions](registering-elements#main-document-definitions).
 {.alert .alert-info}
 
-### Provide a template by defining a template property on the constructor {#templateobject}
-
-As an alternative to specifying the element's template in markup, you can define a static 
-`template` property. 
-
-One way to do this is to specify a string template by creating a static template getter that 
-returns a template literal. 
+### Define a template property on the constructor {#templateobject}
 
-Template literals are denoted by being enclosed in back-ticks (<code>` `</code>). Polymer 2.4+ uses
-the functionality of embedded expressions in template literals to provide convenient ways to 
-[extend inherited templates](#inherited-templates). [Read more about template literals on MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals).
+As an alternative to specifying the element's template in markup, you can define a `template`
+property on the element's constructor. To create a static template getter, override Polymer's
+default `template` getter. This getter is called _once_, when the first instance of the element
+is upgraded.
 
-To create a static template getter, override Polymer's default `template` getter. This getter is
-called _once_, when the first instance of the element is upgraded.
-
-Specify a string template in a static template getter: { .caption }
+The template getter can return either a string or an instance of `HTMLTemplateElement`. Support
+for the string return value is deprecated in Polymer 2.4, and will be removed in 3.0. 
 
 ```js
-class MyElement extends Polymer.Element {
+// Define a template on a new element with a static template getter.
 
-  static get template() {
-    return `<style>:host { color: blue; }</style>
-       <h2>String template</h2>
-       <div>I've got a string template!</div>`
+class NewElement extends Polymer.Element {
+  // Override Polymer's default template getter
+  static get template () {
+    // Polymer 2.x: Return string template or instance of HTMLTemplateElement
+    // Polymer 2.4+: Return HTMLTemplateElement 
   }
 }
-
-customElements.define('my-element', MyElement);
 ```
 
-[See a working example in Plunker](http://plnkr.co/edit/AzkMDJ?p=preview).
+Polymer 2.4+ provides a helper function (`Polymer.html`) to generate an `HTMLTemplateElement`
+instance from a template literal.
```

Comment:
I'd do "JavaScript template literal" to be extra clear, since we're also talking about HTML templates too